webpack/utils/getCSSManifest.js

```diff
@@ -60,7 +60,7 @@ function createManifest() {
   return manifest
 }
 
-export function getCSSManifest() {
+exports.getCSSManifest = () => {
```

Comment:
I forgot that you can use `exports`... such a weird nuance. You just can't use it when assigning, right?

Reply:
Exactly, so it can be a bit wierd... Still better for static analysis tho so decided to go with it.